src/Controller/AuthController.php

```diff
@@ -71,14 +70,10 @@ public function login(Request $request)
             //the messageManager holds all the errorMessages that will be displayed in the view
             $this->messageManager->addErrorMessage($e->getMessage());
 
```

Comment:
Change has nothing to do with commit|PR message.
Compare PR name with method/Controller name